packages/shared/src/AdaptableBadge.js

```diff
@@ -51,4 +60,10 @@ const style = StyleSheet.create({
   text: {
     color: defaultTokens.paletteWhite,
   },
+  info: {
+    backgroundColor: defaultTokens.paletteBlueNormal,
+  },
+  circled: {
+    borderRadius: 11,
```

Comment:
The designer is not using Orbit values?

Reply:
I think you already know the answer to that 😞